modules/reindex/build.gradle

```diff
@@ -37,11 +37,12 @@ run {
 
 dependencies {
   compile "org.elasticsearch.client:rest:${version}"
+  // TODO dm: This is not needed as we pull in dependencies of org.elasticsearch.. projects transitive -> separate PR based on master?
   // dependencies of the rest client
-  compile "org.apache.httpcomponents:httpclient:${versions.httpclient}"
-  compile "org.apache.httpcomponents:httpcore:${versions.httpcore}"
-  compile "commons-codec:commons-codec:${versions.commonscodec}"
-  compile "commons-logging:commons-logging:${versions.commonslogging}"
+  //compile "org.apache.httpcomponents:httpclient:${versions.httpclient}"
+  //compile "org.apache.httpcomponents:httpcore:${versions.httpcore}"
+  //compile "commons-codec:commons-codec:${versions.commonscodec}"
+  //compile "commons-logging:commons-logging:${versions.commonslogging}"
```

Comment:
ok the reason is that httpaasyncclient depends on httpclient 4.5.2 and httpcore 4.4.5, but httpclient 4.5.2 depends on httpcore 4.4.4 :(   that PR still makes sense